### PR TITLE
Implement PoissonVI to model scATAC-seq fragment counts

### LINKED ISSUE #1926: Implement PoissonVI to model scATAC-seq fragment counts
**Is your feature request related to a problem? Please describe.**
PeakVI currently only supports modelling of binarized scATAC-seq data. In our [preprint](https://www.biorxiv.org/content/10.1101/2022.05.04.490536v1), we showed that it can be beneficial to actually model the scATAC-seq fragment counts.

**Describe the solution you'd like**
As previously discussed with @adamgayoso, it would be good to add the PoissonVI model to scvi, that users can also model the fragment count data.

### PR DESCRIPTION
**Pull request by lauradmartens**
## New
* Implements PoissonVI model

## Related Issues
Closes #1926

### CODE REVIEW

File: scvi/data/_manager.py [hunk 1]
```diff
@@ -87,7 +86,7 @@ def __init__(
         self.fields = fields or []
         self.validation_checks = validation_checks or AnnDataManagerValidationCheck()
         self._registry = {
-            _constants._SCVI_VERSION_KEY: scvi.__version__,
+            #      _constants._SCVI_VERSION_KEY: scvi.__version__,
```
Comment (WeilerP) [inline scvi/data/_manager.py:89]: @lauradmartens, any specific reason why you commented this part out? At first glance, this is the reason why the CI currently fails.

File: scvi/model/_poissonvi.py [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+            fdr_target,
+            silent,
+            m1_domain_fn=m1_domain_fn,
+            **kwargs,
+        )
+
+        return result
+
+    @staticmethod
+    def reads_to_fragments(
```
Comment (lauradmartens) [inline scvi/model/_poissonvi.py:453]: @marianogabitto this is the code for the conversion #1926

File: scvi/model/_poissonvi.py [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+        """
+        if layer:
+            data = np.ceil(adata.layers[layer].data / 2)
+        else:
+            data = np.ceil(adata.X.data / 2)
+
+        adata.layers["counts"] = adata.X.copy()
+        adata.layers["counts"].data = data
+
+    def _validate_fragment_counts(adata, layer=None):
```
Comment (lauradmartens) [inline scvi/model/_poissonvi.py:474]: @marianogabitto this is the code for checking that the data is not binarized or read counts #1926

**Comment by martinkim0** [comment]
Closing this in favor of #2249

**Pull request closed without merging**